editor/editor_interface.h

```diff
@@ -110,6 +123,12 @@ class EditorInterface : public Object {
 	String get_current_feature_profile() const;
 	void set_current_feature_profile(const String &p_profile_name);
 
+	// Editor dialogs.
+
+	void popup_dialog_scene_tree(const Callable &p_callback, const TypedArray<StringName> &p_valid_types = TypedArray<StringName>());
+	// Must use Vector<int> because exposing Vector<Variant::Type> is not supported.
+	void popup_dialog_property_selector(Object *p_object, const Callable &p_callback, const Vector<int> &p_type_filter = Vector<int>());
```

Comment:
```suggestion
	void popup_dialog_property_selector(Object *p_object, const Callable &p_callback, const PackedInt32Array &p_type_filter = PackedInt32Array());
```
Better to be explicit here, especially since `int` may differ between platforms.